File: MotivApp/app/src/main/java/com/motiv/piotr/FragmentsPagerAdapter.java
package com.motiv.piotr;

import androidx.fragment.app.*;

public class FragmentsPagerAdapter extends FragmentStatePagerAdapter {

    public FragmentsPagerAdapter(FragmentManager fm) {
        super(fm);
    }

    @Override
    public int getCount() {

        return 3;
    }

    @Override
    public Fragment getItem(int position) {

        switch (position) {
            case 0:
                return new UsersListFragment();
            case 1:
                return new PostsListFragment();
            case 2:
                return new PhotosFragment();
        }
        throw new RuntimeException();
    }

    @Override
    public java.lang.String getPageTitle(int position) {

        switch (position) {
            case 0:
                return "UsersListFragment";
            case 1:
                return "PostsListFragment";
            case 2:
                return "PhotosFragment";
        }
        return "";
    }
}

File: MotivApp/app/src/main/java/com/motiv/piotr/FragmentsPagerAdapter.kt
 
package com.motiv.piotr
import androidx.fragment.app.*
import dagger.*
import dagger.android.*
import dagger.android.support.*
import javax.inject.*

public class FragmentsPagerAdapter(val fm: FragmentManager) : FragmentStatePagerAdapter(fm) {

    override fun getCount(): Int {
        return 3
    } override fun getItem(position: Int): Fragment {
        when (position) {
            0 -> return UsersListFragment()
            1 -> return PostsListFragment()
            2 -> return PhotosFragment()
        }
        throw RuntimeException()
    } override fun getPageTitle(position: Int): String {
        when (position) {
            0 -> return "UsersListFragment"
            1 -> return "PostsListFragment"
            2 -> return "PhotosFragment"
        }
        return ""
    }
}
